packages/animations/lib/src/modal.dart

```diff
@@ -27,6 +29,7 @@ typedef _ModalTransitionBuilder = Widget Function(
 /// modal route that will be displayed, such as the enter and exit
 /// transitions, the duration of the transitions, and modal barrier
 /// properties.
+/// By default, `configuration` is [FadeScaleTransitionConfiguration].
```

Comment:
We usually specify the default at the end of the API doc. Could you shift this to be the last line? Thank you!

Reply:
@shihaohong 

Thanks for the review again 🙏 

But, I don't understand what change is desired 🤔 

I think current `default` position is same as like this:

<img width="660" alt="image" src="https://user-images.githubusercontent.com/1255062/80059162-358fdb00-8566-11ea-8220-7aeabb7202c4.png">

https://github.com/flutter/packages/blob/f5411da6750d5f450f7f4dffa53561e379efa790/packages/animations/lib/src/modal.dart#L157-L167

---

Could you suggest the change by using [multi-line code suggestions](https://github.blog/changelog/2020-02-26-multi-line-code-suggestions-beta/)?